src/sentry/rules/processing/delayed_processing.py

```diff
@@ -142,7 +142,9 @@ def get_condition_group_results(
             return None
 
         _, duration = condition_inst.intervals[unique_condition.interval]
-        comparison_interval = condition_inst.intervals[unique_condition.interval][1]
+        comparison_interval = condition_inst.intervals[
+            condition_data.get("comparisonInterval", "5m")
```

Comment:
The default is not really necessary b/c all percent conditions should have this set, and we only use condition_interval in `get_rate_bulk` if the condition is a percent condition in the first place.

However, it's good to be safe and match what we have for regular processing
https://github.com/getsentry/sentry/blob/598b30da271b9ff83226c0f6d85912e4f3e64af1/src/sentry/rules/conditions/event_frequency.py#L168

Reply:
yeah, nice to include the default. i think extra credit would be to have that default set as a constant. then we can reuse it for the API or other areas of code in the future.